### ISSUE TITLE
Can you edit multiple strings at once?

### ISSUE THREAD

**Issue by asd2123** [leonardosnt/jar-string-editor#2]
Hello thanks for the program,but  can you edit multiple strings at once. For Example
I want to replace the word "Hello" to "Bye" on all the strings.

**Comment by leonardosnt** [comment]
No, it doesn't have this feature. But it would be useful to have, I'll consider implementing this.
